.gitlab/internal_kubernetes_deploy.yml

```diff
@@ -0,0 +1,31 @@
+---
+# internal_kubernetes_deploy stage
+# Contains jobs to trigger a pipeline in our k8s-datadog-agent-ops repo
+
+internal_kubernetes_deploy_experimental:
+  stage: internal_kubernetes_deploy
+  rules:
+  - if: $FORCE_K8S_DEPLOYMENT == "true"
+    when: always
+  - if: $CI_COMMIT_BRANCH != "main"
+    when: never
+  - !reference [.on_deploy_a7]
+  needs:
+  - job: docker_trigger_internal
+    artifacts: false
+  - job: docker_trigger_cluster_agent_internal
+    artifacts: false
+  - job: k8s-e2e-main # Currently only require container Argo workflow
+    artifacts: false
+    optional: true
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/deb_x64:$DATADOG_AGENT_BUILDIMAGES
+  tags: ["runner:main"]
+  variables:
+    OPTION_AUTOMATIC_ROLLOUT: "true"
+    OPTION_PRE_SCRIPT: "patch-cluster-images-operator.sh stormeagle ${CI_COMMIT_REF_SLUG}-jmx clusteragent.image.tag=${CI_COMMIT_REF_SLUG}"
```

Comment:
the tag define is `RELEASE_TAG="${RELEASE_TAG}-${CI_COMMIT_SHORT_SHA}"` but here you use `${CI_COMMIT_REF_SLUG}` is it a mistake?

Reply:
Yes that's indeed correct, it should be `${CI_COMMIT_REF_SLUG}-jmx-${CI_COMMIT_SHORT_SHA}`